invenio_notify/records/models.py

```diff
@@ -117,6 +133,62 @@ def find_review_id_by_user_id(cls, user_id):
         return [r[0] for r in db.session.query(cls.reviewer_id).filter(cls.user_id == user_id).all()]
 
 
+class ReviewerModel(db.Model, Timestamp, DbOperationMixin):
+    __tablename__ = "reviewer"
```

Comment:
__tablename__ = "notify_reviewer"

Reply:
as I said in previous PR, that is unnecessary add prefix notify_ to all table.
reivewer and endorsement will have more sub tables named reivewer_ and endorsement_ in the future
if you check db schema there have a lot of tables names that not start with project name